data/language/en-GB.txt

```diff
@@ -4232,6 +4232,9 @@ STR_5920    :Render weather effects
 STR_5921    :{SMALLFONT}{BLACK}If enabled, rain and gloomy colours will be rendered during storms.
 STR_5922    :{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{SMALLFONT}{BLACK}Max {STRINGID}
 STR_5923    :{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{SMALLFONT}{BLACK}Max {COMMA16} {STRINGID} per train
+STR_5924    :New folder
```

Comment:
`$CWD`, `cd`, `dir`, and just a handful of ["folder"](https://github.com/torvalds/linux/search?utf8=%E2%9C%93&q=folder) vs ["directory"](https://github.com/torvalds/linux/search?utf8=%E2%9C%93&q=directory) in public *filesystem* sources all point to "directory".